app/views/navbar.scala.html

```diff
@@ -535,4 +530,4 @@ <h4 class="modal-title" id="sign-up-label">@Messages("authenticate.signup")</h4>
             @Messages("navbar.signin")
         </a>
     </template>
-}
+}
```

Comment:
Add newline at end of file, as per [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)